pages/api/id/[form]/owners.ts

```diff
@@ -104,7 +104,7 @@ export async function addEmailToForm(req: NextApiRequest, res: NextApiResponse):
     );
     return res.status(200).json({ success: result.rows[0] });
   } catch (error) {
-    const message = error.message;
+    const message = (error as Error).message;
```

Comment:
unnecessary typecasting ?

Reply:
See other comment about `error` in a catch block always being typed `unknown`